dev-docs/bidder-adapter-1.md

```diff
@@ -170,11 +170,28 @@ registerBidder(spec);
 
 When the page asks Prebid.js for bids, your module's `buildRequests` function will be executed. Building the request will use data from several places:
 
-* *AdUnit params*: The arguments provided by the page are in `bidderRequest.bids[]`.
+* *AdUnit params*: The arguments provided by the page are in `validBidRequests` as illustrated below.
 * *Transaction ID*: `bidderRequest.bids[].transactionId` should be sent to your server and forwarded to any Demand Side Platforms your server communicates with.
 * *Ad Server Currency*: If your service supports bidding in more than one currency, your adapter should call `config.getConfig(currency)` to see if the page has defined which currency it needs for the ad server.
 * *Referrer*: Referrer should be passed into your server and utilized there. This is important in contexts like AMP where the original page referrer isn't available directly to the adapter. We suggest using the `utils.getTopWindowUrl()` function to obtain the referrer.
 
+Sample array entry for validBidRequests[]:
+{% highlight js %}
+[{
+  "bidder": "example",
+  "bidId": "51ef8751f9aead",
+  "params": {
+    "cId": "59ac1da80784890004047d89",
+    ...
+  },
+  "adUnitCode": "div-gpt-ad-1460505748561-0",
+  "transactionId": "d7b773de-ceaa-484d-89ca-d9f51b8d61ec",
+  "sizes": [[320,50],[300,250],[300,600]],
```

Comment:
would really like to change this format so each adapter doesn't have to parse. I'll make a PR.

Reply:
@mkendall07 - what needs to be parsed? Should we wait on that change to makes these doc updates?  I'd propose moving forward with this PR and updating again when other updates are available.

Reply:
thinking about changes the format to be easier to digest. Maybe instead we just provide an alternative format: 
`sizesParsed : [{width: 300, height:250}, ...]`